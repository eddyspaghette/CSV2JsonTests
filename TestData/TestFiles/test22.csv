ト!ぇ!ご!₣!ψ!😵
ド!え!さ!℀!ↂ!🍕
ッ!あ!が!ざ!℁!⏳
ミ!ぅ!げ!₡!ℬ!☝
デ!う!こ!₢!Δ!⭐
ツ!ぃ!ぎ!せ!℟!▓
ヅ!い!け!₠!Å!○
