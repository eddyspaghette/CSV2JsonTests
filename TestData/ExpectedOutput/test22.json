[
 {
   "FIELD1": "ト!ぇ!ご!₣!ψ!😵"
 },
 {
   "FIELD1": "ド!え!さ!℀!ↂ!🍕"
 },
 {
   "FIELD1": "ッ!あ!が!ざ!℁!⏳"
 },
 {
   "FIELD1": "ミ!ぅ!げ!₡!ℬ!☝"
 },
 {
   "FIELD1": "デ!う!こ!₢!Δ!⭐"
 },
 {
   "FIELD1": "ツ!ぃ!ぎ!せ!℟!▓"
 },
 {
   "FIELD1": "ヅ!い!け!₠!Å!○"
 }
]